apps/kol-amelamdim/components/upload-file-dialog/UploadFileDialog.tsx

```diff
@@ -51,6 +51,7 @@ export const UploadFileDialog = ({
   const [selectedFile, setSelectedFile] = useState<File | null>(null);
   const [submissionError, setSubmissionError] = useState('');
   const [isUploadingInProcess, setIsUploadingInProcess] = useState(false);
+  const [isUploadButtonDisabled, setIsUploadButtonDisabled] = useState(false);
```

Comment:
that state is redundant..
you can directly pass to the buttons disabled prop the `!uploadValidationError`